### PR TITLE
Handler errors and empty results

### PR DESCRIPTION
**Pull request by snova-luiss**

### CODE REVIEW

File: search_assistant/src/search_assistant.py [hunk 1]
```diff
@@ -304,19 +342,29 @@ def querySerpapi(
...
-        context = []
-        for i, result in enumerate(results):
-            context.append(f'[reference:{i+1}] {result.get("title", "")}: {result.get("snippet", "")}')
-        context = '\n\n'.join(context)
+        try:
+            search = GoogleSearch(params)
+            response = search.get_dict()
+
+            knowledge_graph = response.get('knowledge_graph', None)
+            results = response.get('organic_results', None)
```
Comment (snova-jorgep) [inline search_assistant/src/search_assistant.py:350]: results = response.get('organic_results', None) -> results = response.get('organic_results', [])

**Review by snova-jorgep — CHANGES_REQUESTED** [review]
some minor cahnges

**Review by snova-jorgep — APPROVED** [review]
Thanks everything looking good, approved

**Comment by snova-kwasia** [comment]
Merged thanks @snova-luiss

**Pull request merged**